Makefile

```diff
@@ -230,14 +230,28 @@ evhtests:
 	sudo docker run \
 	-w=/go/src/$(PACKAGE_PATH_AKO) \
 	-v $(PWD):/go/src/$(PACKAGE_PATH_AKO) $(BUILD_GO_IMG) \
-	$(GOTEST) -v -mod=vendor $(PACKAGE_PATH_AKO)/tests/evhtests -failfast
+	$(GOTEST) -v -mod=vendor $(PACKAGE_PATH_AKO)/tests/evhtests -failfast -timeout 0
```

Comment:
will this result in an infinite wait and no timeout of this UT?

Reply:
Yes, but we already have similar timeout for other Ut's so added here as well. The default 10m is no longer enough for this UT since more test cases have been added.

Reply:
got it. thanks